diff -up libksysguard-5.18.3/processui/ksysguardprocesslist.cpp.omv~ libksysguard-5.18.3/processui/ksysguardprocesslist.cpp
--- libksysguard-5.18.3/processui/ksysguardprocesslist.cpp.omv~	2020-03-22 21:34:35.873299913 +0100
+++ libksysguard-5.18.3/processui/ksysguardprocesslist.cpp	2020-03-22 21:34:44.579264835 +0100
@@ -39,6 +39,7 @@
 #include <QStyle>
 #include <QStyledItemDelegate>
 #include <QPainter>
+#include <QPainterPath>
 #include <QLineEdit>
 #include <QToolTip>
 #include <QAbstractItemModel>
